mitodl/mitxpro
# feat: Add platform model and associated fields in Course and Program models

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes https://github.com/mitodl/mitxpro/issues/2698

#### What's this PR do?
Adds a new model named `Platform` for course/program partners. Also, adds foreign key fields in the Course and Program models.

#### How should this be manually tested?

- Visit Django admin and verify that Platforms is listed.
- Add/Edit any course/program and verify that the platform field is visible.
- Verify that platform field is added in the APIs
